src/lib/pickles/wrap_hack.ml

```diff
@@ -23,7 +23,9 @@ open Pickles_types
 
 module Padded_length = Nat.N2
 
-(* Pad up to length 2 by preprending dummy values. *)
+(** Pads a vector with `dummy` element to make it of size 2.
+    Warning: panics if the vector has more than one element.
```

Comment:
This isn't true. This `throws an exception if the vector has more than 2 elements`